core/lexicon/en/tv.inc.php

```diff
@@ -38,19 +38,19 @@
 $_lang['tv_err_save'] = 'An error occurred while saving the TV.';
 $_lang['tv_inuse'] = 'The following document(s) are currently using this TV. To continue with the delete operation click the Delete button otherwise click the Cancel button.';
 $_lang['tv_inuse_template'] = 'The following template(s) are currently using this TV: [[+templates]].<br /><br />Please detach the TV from the template(s) before deleting it.';
+$_lang['is_static_tv_desc'] = 'Use an external file to store the default value for this TV. This may be useful if the default value’s content is particularly lengthy.';
 $_lang['tv_lock'] = 'Restrict Editing';
 $_lang['tv_lock_desc'] = 'Only users with “edit_locked” permissions can edit this TV.';
 $_lang['tv_management_msg'] = 'Manage additional custom TVs for your documents.';
-$_lang['tv_name'] = 'TV Name';
-$_lang['tv_name_desc'] = 'Place the content generated by this TV in a Resource, Template, or Chunk using the following MODX tag: <span class="copy-this">[[*<span class="example-replace-name">'.$_lang['example_tag_tvname'].'</span>]]</span>';
+$_lang['tv_name_desc'] = 'Place the content generated by this TV in a Resource, Template, or Chunk using the following MODX tag: <span class="copy-this">[[*<span class="example-replace-name">'.$_lang['example_tag_tv_name'].'</span>]]</span>';
```

Comment:
Hmm, interesting solution. It brakes translations. Need to somehow remove here concatenation or even maybe rewrite the example.

It suggests to translate such thing
<img width="489" alt="Screen Shot 2022-04-20 at 19 05 38" src="https://user-images.githubusercontent.com/303498/164274363-ec4de18d-84a9-410b-812a-177355f771be.png">

Reply:
Ok, I'll take a look at fixing this later today.

Reply:
It is not your issue as it was backported and probably somebody left it before as well, but definitely, it would be nice to fix.